packages/frontend/src/components/match_card.tsx

```diff
@@ -0,0 +1,36 @@
+export function MatchCard(props: { score: Record<string, number> }) {
+  const [firstKey, secondKey] = Object.keys(props.score);
+  const firstValue = props.score[firstKey];
+  const secondValue = props.score[secondKey];
+  return (
+    <div className="bg-gray-400 text-center w-min rounded-lg text-white overflow-hidden">
+      <div className="flex justify-between whitespace-nowrap text-xl font-bold items-center">
+        <p className="m-2">{firstKey} </p>
+        <p
+          className="ml-2 p-2"
+          style={{
+            backgroundColor: firstValue >= secondValue ? "green" : undefined,
```

Comment:
This should be using tailwind with `bg-p-green` type so its using our primary green colour